adm_program/languages/de.xml

```diff
@@ -927,6 +927,10 @@
     <string name="PRO_WANT_DELETE_PHOTO">Soll das ausgewählte Foto wirklich gelöscht werden?</string>
     <string name="PRO_YEARS">Jahre</string>
     <!-- Phrases only in Relationssmodule -->
+    <string name="REL_EDIT_USER_BACKWARD">Benutzerprofil der rückwärtigen Beziehung bearbeiten</string>
```

Comment:
Sounds strange for me: Might be changed to e.g "Benutzerprofil der verursachenden Beziehung bearbeiten"